Formula/coreos-ct.rb

```diff
@@ -1,8 +1,8 @@
 class CoreosCt < Formula
   desc "Convert a Container Linux Config into Ignition"
-  homepage "https://coreos.com/os/docs/latest/configuration.html"
-  url "https://github.com/coreos/container-linux-config-transpiler/archive/v0.9.0.tar.gz"
-  sha256 "140c2a5bfd2562a069882e66c4aee01290417f35ef0db06e11e74b2ccf52de7f"
+  homepage "https://flatcar-linux.org/docs/latest/provisioning/config-transpiler/"
+  url "https://github.com/flatcar/container-linux-config-transpiler/archive/v0.9.4.tar.gz"
```

Comment:
Which other distros/package managers use this version?

Reply:
@carlocab I'm unaware of ct being bundled with any other package managers? I believe currently the only distro utilizes `ct` is Flatcar since CoreOS reached EOL. Flatcar has maintained this fork in the meantime and is still currently utilizing it even though the previous CoreOS repository was archived.

Reply:
Hello, `ct` is packaged on ArchLinux too (see: https://archlinux.org/packages/community/x86_64/ct/) - the version 0.9.4 is on its way.

Reply:
As far as I know, the Flatcar's fork is the only one active and maintained. :)